loadtest/README.MD

```diff
@@ -0,0 +1,32 @@
+## Load Testing
+
```

Comment:
Can we add load testing under a new `tests` directory? Would help us future proof more root level test folders

Reply:
Restructured in the latest commit :)